torch_geometric/nn/conv/gatv2_conv.py

```diff
@@ -218,8 +218,7 @@ def reset_parameters(self):
             self.lin_edge.reset_parameters()
         if isinstance(self.lin_res, Linear):
             self.lin_res.reset_parameters()
-            if self.lin_res.bias is not None:
```

Comment:
Fail to pass the test for type hints if removing these lines. So still check self.lin_res.bias is not None here.